### DISCUSSION TITLE
Please help? Need small code adjustment to hide time stamps in Google Calendar (Stylus Chrome extension)

### DISCUSSION THREAD

**Discussion by pigtailz23** [openstyles/stylus#1747] [Q&A]
# Bug Report

### Bug Description
I used Stylus in 2019 to remedy an infuriating lack of control over Google Calendar after Stylish turned out to be essentially spyware.

I found a good Stylus theme "Google Calendar: Differentiate Weekends with Color" and then added a teensy code I got somewhere (can't remember or find where because it was 2019 but maybe here) SIMPLY to remove the irritating time stamps that pop up on every single darn calendar item I add.

Today, the time stamp hiding is partially failing. It seems events at 30 mins or less now show the time stamp.

If someone could be so kind to check the code I received in 2019 and fix any errors or add something to make it work?

### Screenshots

<img width="573" alt="calendar times" src="https://github.com/openstyles/stylus/assets/161149095/b3a834dd-4363-4414-bedb-d31337e21d10">

### CSS Code
   /* weekends on main calendar */
.st-bg:nth-child(6),
.st-bg:nth-child(7),
.st-dtitle:nth-child(6),
.st-dtitle:nth-child(7),
/* weekends on week calendar */
.tg-weekend {
background-color: #E6EEF5 !important;
}
.KSxb4d {
font-size: 20px !important;
line-height: 8px !important;
}
.Uit9Se {
height: 55px !important;
}
.MANBAf {
font-size: 12px !important;
}
.zlaSJd {
display: none !important;
}
.A6wOnd {
visibility: hidden !important;
}
.gVNoLb {
visibility: hidden !important;
} 

### System Information

- OS: Mac OS Ventura 13.5.2
- Browser: Google Chrome
- Stylus Version: Don't know which Stylus version I'm using and can't find it.

**Comment by pigtailz23** [comment]
Removed and reinstalled to get latest version of Stylus Chrome Extension 1.5.46. 
Still having same problem.

**Comment by autonomous-agent-7** [comment]


**Reply by pigtailz23** [reply]
Yep. No responses.

**Reply by krystian3w** [reply]
Theoretically, making a recurring public schedule on the calendar could help, maybe someone in 2026 doesn't use a Google account and their calendar and won't blindly look for one similar to yours.

Using the API you can check whether `@autonomous-agent-7` hid something in the HTML comment `<!-- comment -->` (e.g. random spam), Repository/Organisation moderators have the option to edit replies in discussions (They don't have to understand the API to understand what's hidden behind the text "there's nothing to render in this response").

**Reply by pigtailz23** [reply]
Not super interested in making my calendar public but I do thank you for the response. Couldn't get anyone to help so I gave up. I did it the old fashioned way. I just type ".............................." at the end of every calendar entry title to push out the darn times. Thank you though. Have a wonderful day!
